plugins/quote.py

```diff
@@ -146,6 +146,53 @@ def get_quote_by_chan(db, chan, num=False):
     return format_quote(data, num, count)
 
 
+@hook.command('findquote')
+def find_quote(db, text, notice):
+    """[#chan] [nick] <to match> OR [nick] <to match> - get the first quote matching <to match> by [nick] in [#chan]"""
+
+    errstr = "No quotes found"
+    find = re.match(r"(?:(#\S+)\s+)?(\S+)\s+(.*)$", text)
+    if not find:
+        notice(find_quote.__doc__)
+        return
+
+    chan, nick, to_match_raw = find.groups()
+    to_match = "%" + to_match_raw + "%"
```

Comment:
You may wish to escape `_` or `%` characters in the test.

Reply:
I don't see why wildcards shouldn't be allowed. maybe I could support * by replacing it with % and escape the rest?

Reply:
A user most likely would not know about the `_` and `%` wildcards and may wish to search for a quote containing those characters. It isn't a required change, just something I've dealt with before.

Reply:
I'll do the * idea then, it will be more intuitive while leaving wildcards in

Reply:
Perhaps implementing `*` -> `%` and `?` -> `_` to imitate glob syntax would be more familiar to users.

Reply:
I think ? is too common a character, * should be enough.